src/main/java/io/github/cottonmc/cotton/Cotton.java

```diff
@@ -60,8 +60,8 @@ public void onInitialize() {
 				"    }\n" +
 				"}\n"));
 		VirtualResourcePackManager.INSTANCE.addPack(
-				new VirtualResourcePack("test", map),
-				Collections.singleton(ResourceType.CLIENT_RESOURCES)
+				new VirtualResourcePack(new Identifier("cotton", "test"), map),
+				Arrays.asList(ResourceType.values())
```

Comment:
I used this for testing, I'll have to revert this line.

Reply:
I can do it before release.